### PR TITLE
rtmp-services: Update ingest list for Livito.tv

### PR DESCRIPTION
**Pull request by vdavari**
### Description
Add new rtmp service (Livito.tv)

### CODE REVIEW

**Comment by Fenrirthviti** [comment]
Please properly fill out the PR template. In the future, do not just delete it.

Do you work for Livito.tv? Per the [service submission guidelines](https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines) we can only accept new services directly from the service.

File: plugins/rtmp-services/data/package.json [hunk 1]
```diff
@@ -8,4 +8,4 @@
             "version": 204
         }
     ]
-}
+}
```
Comment (RytoEX) [inline plugins/rtmp-services/data/package.json:11]: Do not remove the blank line at the end of files.

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -2434,4 +2450,4 @@
             ]
         }
     ]
-}
+}
```
Comment (RytoEX) [inline plugins/rtmp-services/data/services.json:2453]: Do not remove the blank line at the end of files.

**Review by RytoEX — CHANGES_REQUESTED** [review]

**Review by RytoEX — CHANGES_REQUESTED** [review]
Per our [Service Submission Guidelines](https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines):
> The `rtmp-services` package version needs to be incremented for OBS to know that there has been an update, simply increment the version numbers in `plugins/rtmp-services/data/package.json` by 1.

In the future, please:
1. Do not use merge commits.
2. Do not make PRs from your master branch.
3. Do not delete the PR template.
4. Squash the commits into a single commit.

Regarding the PR template, it is still not filled out correctly. Please actually fill out the template as requested.

**Comment by Fenrirthviti** [comment]
Closing due to lack of response.

**Pull request closed without merging**